collector/benchmarks/helloworld-tiny/0-println.patch

```diff
@@ -0,0 +1,9 @@
+diff --git a/src/main.rs b/src/main.rs
```

Comment:
Remove this file so the `incr-patched` scenario doesn't happen.

Reply:
Other than that, I'd be ok for this to land as is, for a starting point. Or if you want to do the `exclude-profile`/`exclude-scenario` thing too, that would be nice.

Reply:
Good point, thanks!